src/host/ncp_host.cpp

```diff
@@ -91,6 +96,16 @@ void NcpNetworkProperties::GetDatasetPendingTlvs(otOperationalDatasetTlvs &aData
     OTBR_UNUSED_VARIABLE(aDatasetTlvs);
 }
 
+void NcpNetworkProperties::SetMeshLocalPrefix(const otIp6NetworkPrefix &aMeshLocalPrefix)
+{
+    memcpy(mMeshLocalPrefix.m8, aMeshLocalPrefix.m8, sizeof(mMeshLocalPrefix.m8));
```

Comment:
Do we need to somehow sync/pass this to NCP? (or is it only tracking the value from NCP?)

Reply:
It is only tracking the value from NCP. The set method is only called from the `NcpSpinel` to set the value from NCP :)